clang/cmd.go

```diff
@@ -94,7 +94,14 @@ func Cmd(args []string, api *gen.API) error {
 		}
 	}
 
-	if err := api.HandleDirectory(clangCDirectory); err != nil {
+	headerFiles, err := api.HandleDirectory(clangCDirectory)
```

Comment:
It iterates through the given directory and creates headerfile objects for every header file. So I think "parse" does not fit either. Let's just keep "HandleDirectory" for now? Maybe we have an epiphany someday.